templates/views/index.html

```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
         <div class="row">
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
                 <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                    <p class="__shecodetestimp">This Sca mentoring program was like a breathe of fresh air that I needed
+                        since December 2019. I'm so filled as I joined
+                        and has been growing on my learning path (Php/laravel).
+                        In fact my mentor is the best that anyone can have. She's not only good but naturally a guide.
+                        God bless you @FaithAdekogbe.
+                        Thanks shecodeafrica & all the organizers. I'll forever make you guys proud. 🥰
                     </p>
-                    <p class="__shecodetestimp1">Chika Obi<br><span>Web Developer</span></p><br>
+                    <p class="__shecodetestimp1">Onyekachi Peace<br></p><br>
                 </div>
-                <img src="/images/hadijat.jpg" class="__shecodetestimimg">
+                <img src="/images/PeaceAmara.jpg" class="__shecodetestimimg">
             </div>
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
-                <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                <div class="__shecodetestimcard last-child">
+                    <p class="__shecodetestimp">I met lots of people that has been helping in my journey so far,also
```

Comment:
Update this testimonial to:
'Through She Code Africa, I have met lots of people that have been helping in my journey so far. I am also privileged to be part of SCA's Mentoring Program Cohort 1 and I have learnt so much within a short period.'

Also, do note that this testimonial card is still smaller/shorter than the others. They need to be equal, please adjust appropriately after updating the content

Reply:
Okay, noted